README.md

```diff
@@ -163,6 +163,14 @@ Keys:
 
 **Default**: `"- examples"`
 
+If you want to specify individual sketches to be compiled, you can do so by passing a multi-line string (a so called [literal block scalar](https://www.yaml.info/learn/quote.html#literal)), i.e.
```

Comment:
I think this information would be better moved to a much needed general explanation of the usage of the action inputs that accept YAML documents. Otherwise the same information would need to be duplicated for the `platforms`, `libraries`, and `cli-flags` inputs as well (https://github.com/arduino/compile-sketches/issues/73).

The documentation for each of the individual inputs could then link to that single explanation.